### ISSUE TITLE
Pages which `paginate` are excluded from `collections.pages`

### ISSUE THREAD

**Issue by goulvench** [bridgetownrb/bridgetown#807]
I have pages in the `src/_pages/` folder which I list in a navigation partial. I fetch each page using `collections.pages.resources_by_slug` to access its title and URL. This is very practical because Bridgetown automatically gives me the localised title and URL (the website is generated in 2 locales).

However, when I add pagination in the frontmatter of a page, it disappears from `collections.pages`.

**Bridgetown Version**:
bridgetown 1.3.1 "Kelly Butte"

**Comment by goulvench** [comment]
@jaredcwhite Good point! I was far from having 2 pages' worth of posts, so I hadn't noticed.

In the end I ditched pagination, the list page will list all posts, and individual posts have links to the previous/next post in_locale, which doesn't require pagination.

**Comment by jaredcwhite** [comment]
Use case for why this is still relevant: https://github.com/bridgetownrb/bridgetown/pull/987#issuecomment-2755789948

**Comment by jaredcwhite** [comment]
Not sure I can come up with a workaround off the top of my head, but I'll give this some thought.
